featuretools/feature_base/feature_base.py

```diff
@@ -47,6 +51,13 @@ def __init__(
         # initialize if not already initialized
         if not isinstance(primitive, PrimitiveBase):
             primitive = primitive()
+
```

Comment:
What if the user has already instantiated the primitive before creating the Feature? I don't think the series type will get set in that case.

Reply:
If I am understanding correctly: this should be outside the `if-block`, right?

Reply:
Right - we always want to set the series type on the primitive, not just when it is instantiated during Feature creation.